src/component/buttonPanel/buttonPanel.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Button from '../button';
+import styles from './buttonPanel.module.css';
+
+const Calculator = () => (
```

Comment:
#### [OPTIONAL]

- [ ] For naming convention purposes I highly recommend you change this component's name to `ButtonPanel` as required in the project requirements.